.github/workflows/conformance-aks.yaml

```diff
@@ -250,18 +251,11 @@ jobs:
             --helm-set=debug.enabled=true \
             --helm-set loadBalancer.l7.backend=envoy \
             --helm-set tls.secretsBackend=k8s \
-            --azure-resource-group ${{ env.name }}-${{ matrix.location }} \
-            --wait=false \
-            --rollback=false \
-            --config monitor-aggregation=none \
-            --version="
```

Comment:
I guess we no longer need the empty version to trigger development versions because of the way helm CLI mode works..?

Reply:
yeah in the helm mode, you either specify the version, or it'll use the default version.

... i'm not even sure if we needed this empty `--version` flag in the classic mode tbh 🤔

Reply:
My recollection was that in classic mode, if you didn't add an empty version flag, it would try to just use the default version hardcoded into the CLI instead of the local chart version.

Reply:
yeah it's using the local chart, and the image tags match the commit SHA:
```
Image versions         cilium             quay.io/cilium/cilium-ci:d6fb7cb1c5e6b155f704910736bb20252b74a55d: 2
                       hubble-relay       quay.io/cilium/hubble-relay-ci:d6fb7cb1c5e6b155f704910736bb20252b74a55d: 1
                       cilium-operator    quay.io/cilium/operator-generic-ci:d6fb7cb1c5e6b155f704910736bb20252b74a55d: 1
```